lib/pure/os.nim

```diff
@@ -944,17 +944,18 @@ proc rawRemoveDir(dir: string) =
   else:
     if rmdir(dir) != 0'i32 and errno != ENOENT: raiseOSError(osLastError())
 
-proc removeDir*(dir: string) {.rtl, extern: "nos$1", tags: [
+proc removeDir*(dir: string, recursive = true) {.raises: [OSError], rtl, extern: "nos$1", tags: [
```

Comment:
I don't think it's necessary that `{.raises.}` pragma be used here, given that all other functions in this module is not annotated with it.

Reply:
True, I'll remove.